cigaradvisor/blocks/carousel/carousel.css

```diff
@@ -70,24 +73,31 @@
   text-align: center;
   width: 30px;
   height: 50px;
+  align-items: center;
 }
 
 .carousel.block .carousel-arrow .prev-icon,
 .carousel.block .carousel-arrow .next-icon {
-  display: inline-block;
+  display: flex;
   font-size: 45px;
   color: #fff;
 }
 
-.carousel.block .carousel-arrow .next-icon {
-  transform: rotate(180deg);
-  margin: 10%;
+.carousel.block .carousel-arrow .next-icon::before {
+  content: '\f054';
+  font-family: var(--ff-fontawesome);
+  padding-left: 50%;
+  font-size: x-large;
```

Comment:
What does `x-large` mean? Let's be specific - either a variable or a px size (variable preferred if we have one)